.github/workflows/quality.yml

```
@@ -1,6 +1,6 @@
 name: Quality
 
-on: [push]
+on: [push, pull_request]
```

Needs to be improved. 
```suggestion
on: 
  push:
  pull_request:
    branches:
      - main
```